access_point/server/server.py

```diff
@@ -24,7 +26,11 @@ def config():
     tts = request.args.get('tts')
     hotword = request.args.get('hotword')
     wake = request.args.get('wake')
-    subprocess.Popen(['sudo', 'bash', access_point_folder + '/config.sh ', stt, tts, hotword, wake])  #nosec #pylint-disable type: ignore
+    subprocess.Popen(['sudo', 'python3', configuration_folder, stt, tts, hotword, wake])  #nosec #pylint-disable type: ignore
+    subprocess.call(['sudo', 'systemctl', 'daemon-reload']) #nosec #pylint-disable type: ignore
```

Comment:
@prateekiiest , why have you implemented it here?? These enable/disable calls are being done in the respective scripts which are not wrappers eg. in `install.sh`

Reply:
@stealthanthrax the enable/disable calls are being done currently under https://github.com/fossasia/susi_linux/blob/0f4d740d1a8da4afb1da7252ffcc4c6c5f75a29e/access_point/server/config.sh#L14

I am replacing the functionalities under config.sh here